cmd/root.go

```diff
@@ -22,21 +20,14 @@ import (
 // logger is the logger of the application.
 var logger = &util.Logger{}
 
-// env is a map of environment variables.
-var env = util.EnvMap{
-	util.EnvAivenProjectName: "",
-}
-
-// client is a pointer to the Aiven client.
-var client = &aiven.Client{}
-
 // NewCmdRoot returns a pointer to the root command.
 func NewCmdRoot(l *util.Logger) *cobra.Command {
 	cmd := &cobra.Command{
-		Use: "go-api-schemas",
+		Use: "go-api-schemas openapi.json",
```

Comment:
Is `openapi.json` correct here? Does the current version support both openapi.json (and openapi-uc.json) as well as querying the endpoints?